pkg/cmd/gist/create/create.go

```diff
@@ -105,7 +105,7 @@ func createRun(opts *CreateOptions) error {
 	}
 
 	// TODO: GHE support
-	gist, err := apiCreate(httpClient, ghinstance.Default(), opts.Description, opts.Public, files)
+	gist, err := apiCreate(httpClient, ghinstance.OverridableDefault(), opts.Description, opts.Public, files)
```

Comment:
To mark this TODO as done is the missing piece the ability to post gists to a GHE host without needing `GH_HOST`?

Reply:
Do you feel that commands such as `api` and `gist create` should also accept a `--host` parameter, so that all necessary parameters can be passed via arguments?

Reply:
Not urgently; it's something I'm comfortable waiting for a user to request. My current hunch is that people will mostly use `github.com` _or_ their enterprise host and that we don't need flags like that. I was just curious about the TODO.

Reply:
You're right; I think I just forgot to take that TODO comment out. But I'm also wondering whether there should be a `--host` flag for these commands. I'm also comfortable waiting and seeing the community reaction.

For people who mostly just use their GHE and not github.com, we were also considering a configuration option such as `default_host` to select a default hostname as an alternative to `GH_HOST` environment variable. But this feels like it would be better as a follow-up changeset.